patchy_protein/operations.py

```diff
@@ -0,0 +1,287 @@
+import flow
+from flow import environments
+from flow import FlowProject
+import signac
+import numpy as np
+import math
+
+benchmark_name = 'patchy_protein'
+
+# state point parameters
+npatch = 3
+patch_select = [0,1,2]
+eps = 0.33
+phi_p = 0.25
+sigma = 0.25
+V = 6.167 # excluded volume
+
+# helper functions
+def quaternion_from_matrix(T):
```

Comment:
`rowan` has this functionality, but I can see you wanting to avoid the dependency.